pkg/keystore/keystore.go

```diff
@@ -52,13 +53,20 @@ func New(config *Config) *Keystore {
 	return &Keystore{config}
 }
 
-var defaultConfig = &Config{
-	RootPath:         "/etc/rkt/trustedkeys/root.d",
-	PrefixPath:       "/etc/rkt/trustedkeys/prefix.d",
-	SystemRootPath:   "/usr/lib/rkt/trustedkeys/root.d",
-	SystemPrefixPath: "/usr/lib/rkt/trustedkeys/prefix.d",
+func NewConfig(systemPath, customPath string) *Config {
+	tk := "trustedkeys"
```

Comment:
Ok, will do.

I just don't like repeating strings. A mistake in variable name can be caught by compiler, but nothing catches those in strings. :)